src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -377,9 +381,13 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(config : HostConfig<T, P,
     // ref unmounts.
     nextEffect = firstEffect;
     while (nextEffect) {
-      try {
-        commitAllHostEffects(finishedWork);
-      } catch (error) {
+      let error;
+      if (__DEV__) {
+        error = invokeGuardedCallback('commitphase1', commitAllHostEffects, null, finishedWork);
```

Comment:
I'm guessing the `name` parameter is only important for ErrorUtils. I'll come back to this before landing.